botorch/models/transforms/outcome.py

```diff
@@ -631,3 +631,128 @@ def untransform_posterior(self, posterior: Posterior) -> Posterior:
             posterior=posterior,
             sample_transform=lambda x: x.pow(1.0 / self.power),
         )
+
+
+class Bilog(OutcomeTransform):
+    r"""Bilog-transform outcomes.
+
+    Useful for modeling constraining functions. Magnifies values near zero and
+    flattens extreme values outside the domain [-1,1].
```

Comment:
Can you also add a pointer to the SCBO paper here so people know where this transform comes from?